packages/web-component/src/components/read-along-component/read-along.tsx

```diff
@@ -845,6 +854,18 @@ export class ReadAlongComponent {
       "loading": {
         "eng": "Loading...",
         "fra": "Chargement en cours"
+      },
+      "add-line": {
+        "eng": "Add line",
+        "fra": "Ajouter une ligne" 
+      },
+      "remove-line": {
+        "eng": "Remove line",
```

Comment:
This is the wrong place to comment on this, I know, but I found a bug: if I add a translation, click the download button to get my RA.html, then click "Remove Line", the line disappears from the edit interface but if I re-download, the deleted line still shows in the downloaded RA.html.